bgpd/bgp_vty.c

```diff
@@ -2125,6 +2125,35 @@ DEFUN (no_bgp_graceful_restart_preserve_fw,
 	return CMD_SUCCESS;
 }
 
+DEFUN(bgp_refresh_stalepath_time, bgp_refresh_stalepath_time_cmd,
+      "bgp refresh stalepath-time (1-4095)",
+      "BGP specific commands\n"
+      "Route Refresh capability parameters\n"
+      "Set the time to wait to delete stale routes after the timer expires\n"
+      "Delay value (seconds)\n")
+{
```

Comment:
are there any recommendation of refresh stalepath timer value, against graceful stalepath timer value?.

Reply:
Only what RFC defines is:
>In particular,
   these procedures ensure that End-of-RIB (EoR) defined in Graceful
   Restart and EoRR as defined in this specification are kept separate,
   thereby avoiding any premature cleanup of stale routes.

Reply:
ok. when I did that similar implementation on quagga, I reused the graceful restart value to avoid having to configure two timers. I think for a first shot, it is acceptable.

Reply:
Do you suggest reusing graceful timer here?

Reply:
no, I think we have to duplicate timers.

Reply:
I mean set `stalepath` for enhanced route refresh to the same value what is under `bgp graceful-restart stalepath-time`, right?

Reply:
it is ok for me.

Reply:
Even i would agree with @pguibert6WIND on the point of reusing  the value from, bgp graceful-restart stalepath-time.